samcli/commands/init/interactive_init_flow.py

```diff
@@ -229,7 +230,7 @@ def _generate_default_hello_world_application(
     is_package_type_image = bool(package_type == IMAGE)
     if use_case == "Hello World Example" and not (runtime or base_image or is_package_type_image or dependency_manager):
         if click.confirm("\n Use the most popular runtime and package type? (Nodejs and zip)"):
```

Comment:
We forgot to update messaging here, should we use variables here instead of writing them static?